code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -759,7 +759,7 @@
 
 /datum/chemical_reaction/pentaerythritol
 	results = list(/datum/reagent/pentaerythritol = 2)
-	required_reagents = list(/datum/reagent/acetaldehyde = 1, /datum/reagent/toxin/formaldehyde = 3, /datum/reagent/water = 1 )
+	required_reagents = list(/datum/reagent/acetaldehyde = 1, /datum/reagent/toxin/formaldehyde = 3, /datum/reagent/lye = 1)
```

Comment:
accidental change?

Reply:
no it's so that it doesn't instantly react into penta when you make acetaldehyde

Reply:
ah i see

Reply:
This needs better clarification in your PR, in both the CL and comments that it was missed since this is a recipe change

Reply:
done